src/protocol/util/AsyncEventServer.cpp

```diff
@@ -280,9 +280,34 @@ struct AsyncEventServer::implementation : public spl::enable_shared_from_this<im
 
     implementation(std::shared_ptr<boost::asio::io_context>    io_context,
                    const protocol_strategy_factory<char>::ptr& protocol,
+                   const std::string&                          host,
                    unsigned short                              port)
         : io_context_(std::move(io_context))
-        , acceptor_(*io_context_, tcp::endpoint(tcp::v4(), port))
+        , acceptor_([&]() {
+          boost::asio::ip::tcp::endpoint endpoint;
+          if (host.empty()) {
+              endpoint = boost::asio::ip::tcp::endpoint(boost::asio::ip::tcp::v4(), port);
+          } else {
+              try {
+                  auto addr = boost::asio::ip::make_address(host);
+                  // Only allow IPv4 to avoid crashes
+                  if (addr.is_v6()) {
+                      CASPAR_LOG(fatal) << "IPv6 addresses are not supported for host: " << host;
+                      throw std::runtime_error("IPv6 not supported");
+                  }
+                  endpoint = boost::asio::ip::tcp::endpoint(addr, port);
+              } catch (const std::exception& e) {
+                  CASPAR_LOG(fatal) << "Invalid host address: " << host << " - " << e.what();
+                  throw;
+              }
+          }
+          tcp::acceptor a(*io_context_);
+          a.open(endpoint.protocol());
+          a.set_option(boost::asio::ip::tcp::acceptor::reuse_address(true));
+          a.bind(endpoint);
+          a.listen();
+          return a;
```

Comment:
is there a reason for this constructor being more verbose than before? 
Previously it was doing `acceptor_(*io_context_, tcp::endpoint(tcp::v4(), port))`, so it  looks like this could be done with something along the lines of `return tcp::acceptor(*io_context_, endpoint);` instead?